hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/workload/selective/BalancingJob.java

```diff
@@ -48,6 +48,11 @@ public BalancingJob(ConsumerNodesRegistry consumersRegistry,
         this.intervalSeconds = intervalSeconds;
 
         metrics.registerGauge(
+                "consumers-workload." + kafkaCluster + ".selective.all-assignments",
```

Comment:
\[PMD\] INFO: The String literal "consumers\-workload\." appears 4 times in this file; the first occurrence is on line 51